addon/chrome/content/preferences/preferences.css

```diff
@@ -85,6 +85,10 @@
 
   /* ::::: spinbuttons ::::: */
 
+input {
+  font:-moz-field;  
+}
+
```

Comment:
The font doesn't match the text font that we use, please remove it

since you decrease the size of all the inputs, i'd like to add this padding on the start
```css
input[type="number"] {
  padding-inline-start: 3px;
}
```

Reply:
That's the default font for 'input'. In before I have to increase all 'size' to make them show all digits.

Reply:
> ```css
> input[type="number"] {
>   padding-inline-start: 3px;
> }
> ```

And this have no use at all.
![图片](https://user-images.githubusercontent.com/8281345/126078305-1311a2aa-a5c1-4c4f-bf36-b0f7f837758d.png)
Digits are still been covered by withe area.

Reply:
it does on my PC

Reply:
> 
> That's the default font for 'input'. In before I have to increase all 'size' to make them show all digits.

1. you didn't increase all 'size' you reduce all sizes to be the same as maxlength.
since the size only control the width of the input and not the content i set it to be 1 higher then its maxlength for all input items. try to set it back, it should work better.

2. we use ```font: message-box;``` for our prefwindow, the font is smaller then -moz-field, i think that all the font sizes should be the same, i don't mind increasing the default size of the font from 12px to 13.33px